core/src/test/java/io/confluent/rest/SaslTest.java

```diff
@@ -101,7 +102,7 @@ public void setUp() throws Exception {
                   + "};\n";
     Files.write(
         jaasFile.toPath(),
-        jaas.getBytes(),
+        jaas.getBytes(Charset.forName("UTF-8")),
```

Comment:
Might as well just use the string constant directly, using the charset has no benefit (see https://github.com/druid-io/druid/blob/master/java-util/src/main/java/io/druid/java/util/common/StringUtils.java)